File: deus.py
import tkinter as tk
from tkinter import filedialog
import subprocess

def on_submit():
    folder_path = folder_entry.get()
    string_value = string_entry.get()
    if folder_path == "" or string_value == "":
        label.config(text="Por favor, preencha todos os campos", fg="red")
    else:
        subprocess.run(["powershell.exe", ".\\teste.ps1", "-folderPath", folder_path, "-folderName", string_value])
        label.config(text="Arquivos criados com sucesso!", fg="green")

def browse_folder():
    folder_path = filedialog.askdirectory()
    folder_entry.delete(0, tk.END)
    folder_entry.insert(0, folder_path)

root = tk.Tk()
root.geometry("450x150")
root.configure(bg="white")
root.title("Criar Pasta e Arquivos")

folder_label = tk.Label(root, text="Caminho da pasta:", bg="white")
folder_label.pack()
folder_entry = tk.Entry(root, bd=1, relief="solid", width=60)
folder_entry.pack()
folder_button = tk.Button(root, text="Procurar", command=browse_folder, bg="#009688", fg="white", borderwidth=0, width=51)
folder_button.pack()

string_label = tk.Label(root, text="Nome da pasta:", bg="white")
string_label.pack()
string_entry = tk.Entry(root, bd=1, relief="solid", width=60)
string_entry.pack()

submit_button = tk.Button(root, text="Enviar", command=on_submit, bg="#009688", fg="white", borderwidth=0, width=51)
submit_button.pack()

label = tk.Label(root, bg="white")
label.pack()

root.mainloop()

File: deus.cpp
#include <QApplication>
#include <QDir>
#include <QFile>
#include <QFileDialog>
#include <QHBoxLayout>
#include <QLabel>
#include <QLineEdit>
#include <QProcess>
#include <QPushButton>
#include <QVBoxLayout>
#include <QWidget>

class MainWindow : public QWidget {
 public:
  MainWindow(QWidget *parent = nullptr) : QWidget(parent) {
    // Create folder label and entry
    folderLabel = new QLabel("Caminho da pasta:", this);
    folderEntry = new QLineEdit(this);
    folderButton = new QPushButton("Procurar", this);
    folderButton->setStyleSheet("background-color: #009688; color: white;");
    connect(folderButton, &QPushButton::clicked, this, &MainWindow::browseFolder);
    QHBoxLayout *folderLayout = new QHBoxLayout();
    folderLayout->addWidget(folderLabel);
    folderLayout->addWidget(folderEntry);
    folderLayout->addWidget(folderButton);

    // Create string label and entry
    stringLabel = new QLabel("Nome da pasta:", this);
    stringEntry = new QLineEdit(this);
    QHBoxLayout *stringLayout = new QHBoxLayout();
    stringLayout->addWidget(stringLabel);
    stringLayout->addWidget(stringEntry);

    // Create submit button
    submitButton = new QPushButton("Enviar", this);
    submitButton->setStyleSheet("background-color: #009688; color: white;");
    connect(submitButton, &QPushButton::clicked, this, &MainWindow::onSubmit);

    // Create message label
    messageLabel = new QLabel(this);

    // Create main layout
    QVBoxLayout *mainLayout = new QVBoxLayout();
    mainLayout->addLayout(folderLayout);
    mainLayout->addLayout(stringLayout);
    mainLayout->addWidget(submitButton);
    mainLayout->addWidget(messageLabel);

    setLayout(mainLayout);
  }

 private:
  QLabel *folderLabel, *stringLabel, *messageLabel;
  QLineEdit *folderEntry, *stringEntry;
  QPushButton *folderButton, *submitButton;

  void browseFolder() {    
    // Open file dialog to browse for folder
    QString folderPath = QFileDialog::getExistingDirectory(this, "Procurar pasta");
    if (!folderPath.isEmpty()) {
        folderEntry->setText(folderPath);
    }
  }

  void onSubmit() {
    QString folderPath = folderEntry->text();
    QString stringValue = stringEntry->text();

    if (folderPath.isEmpty() || stringValue.isEmpty()) {
        messageLabel->setText("Por favor, preencha todos os campos");
        messageLabel->setStyleSheet("color: red;");
    } else {
        // Run the PowerShell script
        QString script = "teste.ps1";
        QStringList arguments;
        arguments << "-folderPath" << folderPath << "-folderName" << stringValue;
        QProcess::startDetached("powershell.exe", arguments, QDir::currentPath(), &script);

        messageLabel->setText("Arquivos criados com sucesso!");
        messageLabel->setStyleSheet("color: green;");
    }
  }
};

int main(int argc, char *argv[]) {
  QApplication app(argc, argv);

  MainWindow window;
  window.show();

  return app.exec();
}